### PR TITLE
Samples Migration to TeamsFx Toolkit 

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/bot-daily-task-reminder/nodejs/.env [hunk 1]
```diff
@@ -1,3 +1,5 @@
 MicrosoftAppId=
 MicrosoftAppPassword=
-BaseUrl=
\ No newline at end of file
+BaseUrl=
+MicrosoftAppType=
```
Comment (Prithvi-MSFT) [inline samples/bot-daily-task-reminder/nodejs/.env:4]: remove this 2 settings, if not required. Upate teamsapp.local.yml file also

File: samples/bot-file-upload/nodejs/README.md [hunk 2]
```diff
@@ -37,6 +37,20 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 - Microsoft Teams is installed and you have an account
 - [NodeJS](https://nodejs.org/en/)
 - [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+- [Teams Toolkit for VS Code](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) or [TeamsFx CLI](https://learn.microsoft.com/microsoftteams/platform/toolkit/teamsfx-cli?pivots=version-one)
```
Comment (Prithvi-MSFT) [inline samples/bot-file-upload/nodejs/README.md:40]: Why this new file UserAttachment1.png is added ?

File: samples/bot-request-approval/nodejs/.env [hunk 3]
```diff
@@ -1,3 +1,5 @@
 MicrosoftAppId=
 MicrosoftAppPassword=
-BaseUrl=
\ No newline at end of file
+BaseUrl=
+MicrosoftAppType=
```
Comment (Prithvi-MSFT) [inline samples/bot-request-approval/nodejs/.env:4]: Remove extra app setrtings and update the same in localyml file

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine

**Review by formulahendry — APPROVED** [review]

File: samples/bot-daily-task-reminder/nodejs/.env (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/bot-daily-task-reminder/nodejs/.env]: fixed

File: samples/bot-file-upload/nodejs/README.md (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/bot-file-upload/nodejs/README.md:40]: fixed

File: samples/bot-request-approval/nodejs/.env (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/bot-request-approval/nodejs/.env]: fixed

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Kindly run build pipeline and share the link.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**